### ISSUE TITLE
Table with attribute named "default" causes exception

### ISSUE THREAD

**Issue by edp-penso** [ptmcg/littletable#1]
If I have a table with an attribute named "default", `littletable` throws an error when calling `present()`

Example code:

```
from types import SimpleNamespace
from littletable import Table

table = Table('example')
table.insert(SimpleNamespace(
    name='Item 1',
    default=1234,
))
table.present()
```

Exception raised:
```
TypeError: 'NoneType' object is not iterable
```

If I change the name `default`, it works as expected.

Great library, thanks very much!

**Comment by ptmcg** [comment]
Shipped in 1.4.1 release

**Comment by edp-penso** [comment]
Thanks!
